.circleci/config.yml

```diff
@@ -3,15 +3,15 @@ jobs:
   test:
     working_directory: ~/circle
     docker:
```

Comment:
`&ecr_base_image`

Reply:
Does it work with more options? The docker block has the environment options too. Let me try.

Reply:
ah yes, sorry didn't spot that. no i don't think it does